elasticai/creator/nn/fixed_point/conv1d/simulation_test.py

```diff
@@ -0,0 +1,123 @@
+"""
+1. create testbench
+2. save testbench to build folder
+3. hand sequence of vhdl files in correct order to simulation tool
+4. hand input data for testcase to simulation tool
+5. let simulation tool compile files (if necessary)
+6. let simulation tool run the simulation
+7. parse/deserialize simulation output to required data
+"""
```

Comment:
I guess we can remove this comment, right?

Reply:
Remove it.